src/Identity/v3/Service.php

```diff
@@ -423,6 +423,21 @@ public function createPolicy(array $options): Models\Policy
     }
 
     /**
+     * Updates a policy object according to the provided options.
+     *
+     * @param string $id
+     * @param array $options {@see \OpenStack\Identity\v3\Api::patchPolicy}
+     *
+     * @return Models\Policy
+     */
+    public function updatePolicy(string $id, array $options): Models\Policy
```

Comment:
this is not how operations are done. the user retrieves a model then performs the update on that object.